docs/integrations/aws-s3/discovery.md

```diff
@@ -6,42 +6,58 @@ sidebar_label: Discovery
 description: Automatically discovering catalog entities from an AWS S3 Bucket
```

Comment:
I was wondering whether this should be renamed from Discovery to Ingestion 🤔

Reply:
hm this type of info page for all integrations are (historically) named discovery, so let's change that separately some other time